README.md

```diff
@@ -134,8 +134,9 @@ You can find a guide on [how to write documentation](docs/README.md).
 Setting up Artemis in your development environment or a demo production environment is really easy following the instructions on https://docs.artemis.cit.tum.de/dev/setup. When you want to support programming exercises, we recommend using the [Integrated Code Lifecycle](https://docs.artemis.cit.tum.de/dev/setup/#integrated-code-lifecycle-setup).
 
 Artemis can also be set up in conjunction with external tools for version control and continuous integration:
-1. [GitLab and Jenkins](https://docs.artemis.cit.tum.de/dev/setup/#jenkins-and-gitlab-setup)
-2. [GitLab and GitLab CI (experimental)](https://docs.artemis.cit.tum.de/dev/setup/#gitlab-ci-and-gitlab-setup)
+1. [LocalVC and Jenkins](https://docs.artemis.cit.tum.de/dev/setup/#jenkins-and-localvc-setup)
+2. [GitLab and Jenkins](https://docs.artemis.cit.tum.de/dev/setup/#jenkins-and-gitlab-setup) (deprecated)
```

Comment:
```suggestion
2. [GitLab and Jenkins (deprecated)](https://docs.artemis.cit.tum.de/dev/setup/#jenkins-and-gitlab-setup)
```
nitpick: to match the `experimental` below

Reply:
Are `deprecated` and `experimental` really the same? I would say no. For me `deprecated` is nothing old that will be dropped in a future version and `experimental` is something new that is not stable enough to use it in a productive environment yet, but will may become stable in future version.

Based on this definitions, I think `deprecated` is correct word here. For GitLab and GitLab CI setup, I think that `experimental` is or was correct. I do not know what are the plans for this. Because it is experimental, it may, will be removed without any migration?

Reply:
I would leave the `deprecated` as in the suggestion. The suggestion was only about changing the formatting to have the parenthesised annotation as part of the link in both cases.